Defs/Ammo/Generic/Generics_Misc.xml

```diff
@@ -5,7 +5,7 @@
 	<!-- ==================== Taurus Judge ========================== -->
 	<CombatExtended.AmmoSetDef>
 		<defName>AmmoSet_RevolverShotgun</defName>
-		<label>revolver shotgun</label>
+		<label>pistol</label>
```

Comment:
This doesn't really "fix" the issue so much as masks it (though we might consider it an acceptable remedy if we conclude the issue is otherwise harmless). The crux of the issue seems to be that our generic ammo system isn't being careful enough when fetching projectiles, so it's just looking for a patching projectile deNamef w/o confirming the ammosetdef it's getting it from.

Reply:
Should I revert the label back to "revolver shotgun" or do we keep it as "pistol" until the root problem is fixed?